src/Symfony/Component/Process/Process.php

```diff
@@ -65,6 +65,14 @@ class Process
     private $latestSignal;
 
     private static $sigchild;
+    private static $posixSignals = array(
```

Comment:
I suggest adding comment near each of them with their name, to make it more maintainable (similar to comments you added in tests when using such value)